### PR TITLE
fix(typo): Small typos in properties description

### PR DESCRIPTION
**Pull request by lordrip**
# Description

This commit fixes a couple of typos:
* Beans: Sets a reference to an _exiting_ bean to use, which is looked up from the registry
* OnException: Turn on exponential _backk_ off

In addition to that, a missing `enum` annotation was added to the LogLevel of `retriesExhaustedLogLevel`

# Target

- [X] I checked that the commit is targeting the correct branch (note that Camel 3 uses `camel-3.x`, whereas Camel 4 uses the `main` branch)

# Tracking
- [X] If this is a large change, bug fix, or code improvement, I checked there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).
[CAMEL-20462](https://issues.apache.org/jira/projects/CAMEL/issues/CAMEL-20462)
fix: [#865](https://github.com/KaotoIO/kaoto-next/issues/865)

# Apache Camel coding standards and style

- [X] I checked that each commit in the pull request has a meaningful subject line and body.

- [X] I have run `mvn clean install -DskipTests` locally and I have committed all auto-generated changes

### CODE REVIEW

**Comment by lordrip** [comment]
I have doubts regarding the autogenerated changes, are the following files intended?:

* `bom/camel-bom/pom.xml`
* components/camel-debezium/camel-debezium-oracle/src/generated/java/org/apache/camel/component/debezium/configuration/OracleConnectorEmbeddedDebeziumConfiguration.java
* components/camel-sql/src/generated/java/org/apache/camel/component/sql/stored/template/generated/SSPTParserTokenManager.java

**Review by oscerd — APPROVED** [review]

File: catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/models/redeliveryPolicy.json [hunk 1]
```diff
@@ -17,12 +17,12 @@
...
-    "retriesExhaustedLogLevel": { "index": 9, "kind": "attribute", "displayName": "Retries Exhausted Log Level", "label": "advanced", "required": false, "type": "object", "javaType": "org.apache.camel.LoggingLevel", "deprecated": false, "autowired": false, "secret": false, "defaultValue": "ERROR", "description": "Sets the logging level to use when retries have been exhausted" },
-    "retryAttemptedLogLevel": { "index": 10, "kind": "attribute", "displayName": "Retry Attempted Log Level", "required": false, "type": "object", "javaType": "org.apache.camel.LoggingLevel", "deprecated": false, "autowired": false, "secret": false, "defaultValue": "DEBUG", "description": "Sets the logging level to use for logging retry attempts" },
+    "retriesExhaustedLogLevel": { "index": 9, "kind": "attribute", "displayName": "Retries Exhausted Log Level", "label": "advanced", "required": false, "type": "enum", "javaType": "org.apache.camel.LoggingLevel", "enum": [ "TRACE", "DEBUG", "INFO", "WARN", "ERROR", "OFF" ], "deprecated": false, "autowired": false, "secret": false, "defaultValue": "ERROR", "description": "Sets the logging level to use when retries have been exhausted" },
```
Comment (lhein) [inline catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/models/redeliveryPolicy.json:24]: You changed the default for `retriesExhaustedLogLevel` from `DEBUG` to `ERROR`. Is that intended?

File: components/camel-sql/src/generated/java/org/apache/camel/component/sql/stored/template/generated/SSPTParserConstants.java [hunk 2]
```diff
@@ -1,67 +1,67 @@
...
-    "<PROCEDURE_BEGIN>",
-    "<PROCEDURE_END>",
-    "<SIMPLE_EXP_TOKEN>",
-    "<PARAMETER_POS_TOKEN>",
-    "<PARAMETER_NAME>",
-    "<IDENTIFIER>",
-  };
-
-}
+/* Generated By:JavaCC: Do not edit this line. SSPTParserConstants.java */
```
Comment (lhein) [inline components/camel-sql/src/generated/java/org/apache/camel/component/sql/stored/template/generated/SSPTParserConstants.java:1]: What changed in that file? Maybe a code formatting thing?

File: catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/models/redeliveryPolicy.json (hunk 1 above)
Comment (lhein) [reply catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/models/redeliveryPolicy.json:24]: ah I see the difference to https://github.com/apache/camel/pull/13312/files#diff-df860c9ccdcd78a71b9e096d18c329d0559d700d40c4cda5be053025444df3cf so this seems fine then.

**Comment by lordrip** [comment]
> I have doubts regarding the autogenerated changes, are the following files intended?:
> 
>     * `bom/camel-bom/pom.xml`
> 
>     * components/camel-debezium/camel-debezium-oracle/src/generated/java/org/apache/camel/component/debezium/configuration/OracleConnectorEmbeddedDebeziumConfiguration.java
> 
>     * components/camel-sql/src/generated/java/org/apache/camel/component/sql/stored/template/generated/SSPTParserTokenManager.java

It turns out that it was a line termination change, I've updated the PR

**Review by orpiske — APPROVED** [review]
LGTM, but ... maybe, it would be great to write the commit messages in the following format:

```
CAMEL-20462: small typos in properties description

<longer description here>
```


That way, it gets linked automatically in the Jira ticket. It makes our life much easier ...

**Comment by lordrip** [comment]
> LGTM, but ... maybe, it would be great to write the commit messages in the following format:
> 
> ```
> CAMEL-20462: small typos in properties description
> 
> <longer description here>
> ```
> 
> That way, it gets linked automatically in the Jira ticket. It makes our life much easier ...

Absolutely, I'll rename the commit, just a sec 👌

**Comment by lordrip** [comment]
I redid the changes using Fedora instead, so now the diff looks cleaner 👍 , sorry for the noise.

**Review by davsclaus — APPROVED** [review]

**Review by orpiske — APPROVED** [review]

**Pull request merged**